<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>619</width>
    <height>374</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>581</width>
     <height>297</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="title">
      <property name="text">
       <string>Title</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="title_le"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="sort">
      <property name="text">
       <string>Sort of coffee</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="dor_label">
      <property name="text">
       <string>Degree of roast</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="vop_label">
      <property name="text">
       <string>Volume of pack (mg)</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="dor_cb">
      <item>
       <property name="text">
        <string>1 (rare)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2 (medium)</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3 (well done)</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QSpinBox" name="vop_sb">
      <property name="maximum">
       <number>1000000</number>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QRadioButton" name="whole">
      <property name="text">
       <string>Whole</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">gow_bg</string>
      </attribute>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="sort_le"/>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="taste">
      <property name="text">
       <string>Taste</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="taste_le"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="price">
      <property name="text">
       <string>Price</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QSpinBox" name="price_sb">
      <property name="maximum">
       <number>10000000</number>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QRadioButton" name="ground">
      <property name="text">
       <string>Ground</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">gow_bg</string>
      </attribute>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QRadioButton" name="mixed">
      <property name="text">
       <string>Mixed</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">gow_bg</string>
      </attribute>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="gow_label">
      <property name="text">
       <string>Ground or whole</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="confirm_pb">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>330</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="gow_bg"/>
 </buttongroups>
</ui>
